update readme (#11)
rsksmart/rsk-starter-box: README.md

Go to [Node.js](https://nodejs.org/en/) if you need to install it.

2. Truffle

Install Truffle globally

```shell
npm install -g truffle
```

## Installation

1. Create a new folder. 
For example, create the folder `rsk-starter`.
Navigate to the folder in the terminal.

```shell
mkdir rsk-starter
cd rsk-starter
```

2. Run the unbox command. This will install all necessary dependencies.

```shell
truffle unbox rsksmart/rsk-starter-box
```

## Development console

Truffle has an interactive console that also spawns a development blockchain. This is very useful for compiling, deploying and testing locally.

3. Run the development console.

```shell
truffle develop
```

4. Take a look at the smart contract `SimpleStorage.sol`. You can check it out in folder `contracts`.

This smart contract has:

* A variable `storedData` to store a number
* A function `get()` to return the number stored at variable `storedData`
* A function `set()` to change the number stored at variable `storedData`

5. Compile and migrate the smart contract. Note inside the development console we don't preface commands with truffle.

```javascript
compile
migrate
```

6. Running contract tests. 

> Our box also comes with the file `TestSimpleStorage.js` for testing the smart contract. You can check it out in `test` folder.

Reviewer: ```suggestion
Our box also comes with the file `TestSimpleStorage.js` for testing the smart contract. You can check it out in the `test` folder.
```

```javascript
test
```

**NOTE**: This box is the starting point for the RSK tutorial [Using rsk-starter-box](https://developers.rsk.co/tutorials/truffle-boxes/rsk-starter-box/).

## Using RSK networks

> Truffle makes developing on RSK easier because we can configure custom networks for RSK. The networks are already configurated in `truffle-config.js` file. 

Reviewer: ```suggestion
Truffle makes developing on RSK easier because we can configure custom networks for RSK. The networks are already configured in the `truffle-config.js` file. 
```

4. Compile and migrate the smart contracts. Note that inside the development console, we don't preface commands with truffle.

    ```javascript
    compile
    migrate
    ```

> 5. Another way to do it is out of truffle console, directly in terminal.

Reviewer: ```suggestion
5. Another option is to run the below commands directly in the terminal, out of the truffle console.
```

    ```shell
    truffle compile
    truffle migrate
    ```

## Next steps

- **Go to tutorial**

> Go to the tutorial [Using rsk-starter-box](https://developers.rsk.co/tutorials/truffle-boxes/rsk-starter-box/) to learn how to interact with `SimpleStorage.sol`. Also we cover all steps with more details, explanations and images.

Reviewer: ```suggestion
Go to the tutorial [Using rsk-starter-box](https://developers.rsk.co/tutorials/truffle-boxes/rsk-starter-box/) to learn how to interact with `SimpleStorage.sol`. Also, we covered all the steps with more details, explanations, and images.
```
